packages/sit-onyx/src/components/OnyxShortcut/OnyxShortcut.stories.ts

```diff
@@ -0,0 +1,78 @@
+import type { Meta, StoryObj } from "@storybook/vue3-vite";
+import OnyxShortcut from "./OnyxShortcut.vue";
+
+const meta: Meta<typeof OnyxShortcut> = {
+  title: "Feedback/Shortcut",
```

Comment:
```suggestion
  title: "Basic/Shortcut",
```

We intended to move the shortcut component to the "Basic" category as its use-case can vary so we found "Basic" the most appropriate one. What do you think?